doc/connectivity/bluetooth/api/mesh/sar_cfg.rst

```diff
@@ -0,0 +1,238 @@
+Both incoming and outgoing messages allocate segments at the start of the transaction, but release them at the different time.
+While the outgoing segmented message releases its segments one by one as soon as they are acknowledged by the receiver, the incoming message will not
+release the segments until the message is fully received.
+Keep this in mind when defining the size of the buffers.
+
+SAR does not impose extra overhead on the access layer payload per segment.
+
+Segmentation and reassembly (SAR) Configuration models are foundation models defined by the
+`Bluetooth Mesh Protocol Specification version 1.1 <https://www.bluetooth.com/specifications/mesh-specifications/>`_.
+These models support the configuration of SAR behavior for nodes in a mesh network.
+
+The SAR Configuration models are:
+
+.. toctree::
+   :maxdepth: 1
+
+   sar_cfg_cli
+   sar_cfg_srv
+
+Intervals, timers and counters
+******************************
+
+Transmission of segments is separated by a segment transmission interval (see the `SAR Segment Interval Step`_ state).
+Other time intervals and delays available for the segmentation and reassembly are:
+
+* Interval between unicast retransmissions (see the states `SAR Unicast Retransmissions Interval Step`_ and `SAR Unicast Retransmissions Interval Increment`_).
+* Interval between multicast retransmissions (see the `SAR Multicast Retransmissions Interval Step`_ state).
+* Segment reception interval (see the `SAR Receiver Segment Interval Step`_ state).
+* Acknowledgment delay increment (see the `SAR Acknowledgment Delay Increment`_ state)
+
+When the last segment marked as unacknowledged is transmitted, the lower transport layer starts a retransmissions timer.
+The initial value of the SAR Unicast Retransmissions timer depends on the value of the TTL field of the message.
+If the TTL field value is greater than ``0``, the initial value for the timer is set according to the following formula::
+
+[unicast retransmissions interval step + unicast retransmissions interval increment * (TTL - 1)]
+
+If the TTL field value is ``0``, the initial value of the timer is set to the unicast retransmissions interval step.
+
+The initial value of the SAR Multicast Retransmissions timer is set to the multicast retransmissions interval.
+
+When the lower transport layer receives a segment of a segmented message, it starts an SAR Discard timer.
+The discard timer tells how long the lower transport layer waits before discarding the segmented message the segment belongs to.
+The initial value of the SAR Discard timer is the discard timeout value indicated by the `SAR Discard Timeout`_ state.
+
+SAR Acknowledgment timer holds the time before a Segment Acknowledgment message is sent for a received segment.
+The initial value of the SAR Acknowledgment timer is calculated using the following formula::
+
+[min(SegN + 0.5 , acknowledgment delay increment) * segment reception interval]
```

Comment:
You should consider using actual math formulas as they render much more nicely.

```suggestion
The initial value of the SAR Acknowledgment timer is calculated using the following formula:

.. math::

   \min(\text{SegN} + 0.5, \text{acknowledgment delay increment}) \times \text{segment reception interval}
```

Alternatively, this simplified form works too

```suggestion
The initial value of the SAR Acknowledgment timer is calculated using the following formula:

.. math::

   min(SegN + 0.5 , acknowledgment~delay~increment) * segment~reception~interval
```

Finally, it's also possible to use the `:math:` role if you want the formula in-line with the text (``blah blah :math:`my formula` blah blah``).

Reply:
FWIW I would recommend you use the math directive everywhere, not just in this particular instance.
Rendered doc available at https://builds.zephyrproject.io/zephyr/pr/61886/docs/connectivity/bluetooth/api/mesh/sar_cfg.html , in case you are not aware and/or are not testing locally.

Reply:
Thanks Benjamin :)
Yeah, I know how they render. Personally I prefer code block rendition over the one using math, it's just something stylistically off with that one for me, for some reason I cannot quite explain. That's why I've been holding back.
But, I'll give it a go and see if it starts growing on me LOL